.gitignore

```diff
@@ -1,3 +1,9 @@
+#Vim swap files
+.*.sw[l-p]
```

Comment:
Editor-specific ignores should probably just go in your ~/.gitignore` (paired with something like `git config --global core.excludesfile ~/.gitignore`).